agency/usr/emiso_engine/daemon/image.cpp

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Same comment as before, if we could call emisoimage or emiso-image or whatever a bit more emiso-related :-)

Reply:
The Image class is part of the "emiso::daemon" name space. It is not enough ?